src/main/java/com/ghinaglam/ghinaglam/repository/ClientRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.ghinaglam.ghinaglam.repository;
+
+import com.ghinaglam.ghinaglam.model.Client;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface ClientRepository extends JpaRepository<Client, Long> {
+    Client findByEmail(String email);
+    void deleteById(Long id);
```

Comment:
You dont need to add this since it a built-in method in JpaRepository

Reply:
Alright, noted